app/helpers/pickup_libraries_helper.rb

```diff
@@ -20,35 +22,70 @@ def label_for_pickup_libraries_dropdown(pickup_libraries)
 
   private
 
-  def select_for_multiple_libraries(form, pickup_libraries)
-    return unless pickup_libraries.length > 1
+  # Symphony
```

Comment:
**suggestion**: this comment isn't clear. Can you expand it a bit?

Reply:
Since this has been changed since, I think it was just to indicate the following line needed to be executed for the Symphony condition, not the FOLIO one.